lib/rubygems/test_case.rb

```diff
@@ -807,9 +807,6 @@ def util_spec(name, version = 2, deps = nil, *files) # :yields: specification
   # location are returned.
 
   def util_gem(name, version, deps = nil, &block)
-    # TODO: deprecate
```

Comment:
Any idea why this was marked to deprecate?

Reply:
I have no idea tbh, but it seems it's not raised across the entire test suite anymore